pkg/api/types.go

```diff
@@ -2226,7 +2226,10 @@ type ListOptions struct {
 	FieldSelector fields.Selector
 	// If true, watch for changes to this list
 	Watch bool
-	// The resource version to watch (no effect on list yet)
+	// For watch, it's the resource version to watch.
+	// For list, if unset (or zero), there is basically no guarantee on what List will return;
```

Comment:
The return should include a resource version which you can still use against a watch and not miss anything. Additionally, this depends on the quorum read flag. If it is on, then this comment is wrong.

@wojtek-t